noir-projects/noir-protocol-circuits/crates/types/src/abis/public_circuit_public_inputs.nr

```diff
@@ -48,7 +48,7 @@ struct PublicCircuitPublicInputs{
 
     prover_address: AztecAddress,
 
-    reverted: bool,
+    reverted: Field,
```

Comment:
Because the base rollup computes its content commitment based on accumulating `Field`s, I felt this was appropriate for a first implementation. Ideally we will trim down the size, but that will require specialization of `compute_tx_effects_hash`